commons-lib/src/test/java/org/opencb/commons/utils/DataModelUtilsTest.java

```diff
@@ -0,0 +1,34 @@
+package org.opencb.commons.utils;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class DataModelUtilsTest {
+
+
+    @Test
+    public void unformattedDataModelJsonString() throws Exception {
+        String dataFieldDocJSON = DataModelsUtils.dataModelToJsonString(Class.forName("org.opencb.commons.utils.TestBeanClass"), false);
+        System.out.println("JSON:\n" + dataFieldDocJSON);
+        Assert.assertFalse(dataFieldDocJSON.contains("\\n"));
+        Assert.assertTrue(isJSONValid(dataFieldDocJSON));
+    }
+
+    public boolean isJSONValid(String test) {
```

Comment:
This could be checked using ObjectMapper library instead. Something like:
`ObjectMapper objectMapper = new ObjectMapper();
 Map map = objectMapper.readValue(test, Map.class);`
If the parsing works, the json is valid.